src/test/java/LoginTests.java

```diff
@@ -1,50 +1,16 @@
-import org.openqa.selenium.By;
-import org.openqa.selenium.WebElement;
-import org.testng.Assert;
 import org.testng.annotations.Test;
 import pages.HomePage;
 import pages.LoginPage;
 
 public class LoginTests extends BaseTest {
-
-    //    @Test(enabled = true, priority = 0, description = "LoginEmptyEmailPasswordTest")
-    @Test(enabled = false, dataProvider = "incorrectLoginProviders", dataProviderClass = BaseTest.class)
-    public void loginEmptyEmailPasswordTest (String email, String password) {
-        LoginPage loginPage = new LoginPage(getDriver());
-
-        loginPage.provideEmail("demo@class.com");
-        loginPage.providePassword("te$t$tudent");
-        loginPage.clickSubmitBtn();
-        Assert.assertEquals(getDriver().getCurrentUrl(), url);
-    }
-
-    //Page Object Model example
-    @Test(enabled = true, priority = 1, description = "Login with Valid Email and Password Test")
-    public void LoginValidEmailPasswordTest () {
-
-        LoginPage loginPage = new LoginPage(getDriver());
-        HomePage homePage = new HomePage(getDriver());
-
-        loginPage.provideEmail("demo@class.com")
-                .providePassword("te$t$tudent")
-                .clickSubmitBtn();
-
-        Assert.assertTrue(homePage.getUserAvatar().isDisplayed());
-
-    }
-
-    @Test(enabled = true, priority = 2, description = "Login with Invalid Email and Valid Password Test")
-    public void loginInvalidEmailValidPasswordTest () {
-        LoginPage loginPage = new LoginPage(getDriver());
-
-        loginPage.provideEmail("invalid@class.com");
-        loginPage.providePassword("te$t$tudent");
-        loginPage.clickSubmitBtn();
-
-        Assert.assertEquals(getDriver().getCurrentUrl(), url);
-
-    }
-
-
-
-}
+    @Test
+    public void LoginPageValidEmailPasswordTest() throws InterruptedException {
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+        loginPage.login();
+        homePage.addPlaylist();
+        homePage.clickNewPlaylist();
+        homePage.namePlaylist();
+        homePage.doubleClickToPlaylist();
+        homePage.newPlaylistName();
+        }
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/215331287-f6510839-a68c-4b78-9f60-8216c5d767a1.png)

Check for Balanced Brackets in an code (well-formedness) {}

For example 

```
public class LoginTests extends BaseTest {
    @Test
    public void loginPageValidEmailPasswordTest() {
        LoginPage loginPage = new LoginPage(driver);
        HomePage homePage = new HomePage(driver);
        loginPage.login();
        homePage.addPlaylist();
        homePage.clickNewPlaylist();
        homePage.namePlaylist();
        homePage.doubleClickToPlaylist();
        homePage.newPlaylistName();
    }
}
```